lib/commands/deals.js

```diff
@@ -66,14 +68,16 @@ module.exports = async (msg, game, ignoredSellers) => {
         {
           name: 'Historical Low',
           value: historicalLow
-        },
-        {
-          name: 'Metacritic User Score',
-          value: metacriticUrl ? `[${metacriticScore}](${metacriticUrl})` : metacriticScore
         }
+
       ],
       timestamp: new Date()
     });
+    steamReview && embed.addFields(
+      {
+        name: 'Steam Review',
+        value: steamReview
+      });
```

Comment:
If short circuit logic is multiline, it's better to expand it into a full if statement to avoid strange formatting. Here the object is "hanging".